src/ConnectedMode/Migration/Wizard/MigrationWizardWindow.xaml.cs

```diff
@@ -53,30 +54,32 @@ public sealed partial class MigrationWizardWindow : DialogWindow, IProgress<Migr
             IConnectedModeMigration connectedModeMigration,
             Action onShowHelp,
             Action onShowTfvcHelp, // null = don't show the Tfvc info section
+            Action onShowSharedBinding,
+            bool isUnderGit,
             ILogger logger)
         {
             this.oldBinding = oldBinding;
             this.connectedModeMigration = connectedModeMigration;
             this.logger = logger;
             this.onShowHelp = onShowHelp;
             this.onShowTfvcHelp = onShowTfvcHelp;
+            this.onShowSharedBinding = onShowSharedBinding;
             threadHandling = ThreadHandling.Instance;
 
             cancellationTokenSource = new CancellationTokenSource();
 
             InitializeComponent();
 
-            SetTfvcInfoVisibility();
+            SetVisibilities(isUnderGit);
 
             this.Closing += OnClosing;
             dialogResult = false;
         }
-        
-        private void SetTfvcInfoVisibility()
-        {
-            bool shouldShowTfvcText = onShowTfvcHelp != null;
 
-            tfvcInfo.Visibility = shouldShowTfvcText ? Visibility.Visible : Visibility.Hidden;
+        private void SetVisibilities(bool isUnderGit)
+        {
+            tfvcInfo.Visibility = isUnderGit ? Visibility.Hidden : Visibility.Visible;
+            chk_SaveSharedBinding.Visibility = isUnderGit ? Visibility.Visible : Visibility.Hidden;
```

Comment:
The `chk_` prefix doesn't look great and adds little value

Reply:
This is a convention using winforms it relays that it's a control it would be easier when reading the code. But for the rest of the window convention is, not having the "_" (e.g. btnPage2) so I'll remove it here also.